src/tests/tracing/tracevalidation/inducedgc/InducedGC.cs

```diff
@@ -1,70 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System;
-using System.IO;
-using Tracing.Tests.Common;
-using Microsoft.Diagnostics.Tracing;
-using Microsoft.Diagnostics.Tracing.Parsers;
-using Microsoft.Diagnostics.Tracing.Parsers.Clr;
-
-namespace Tracing.Tests
-{
-    public static class TraceValidationInducedGC
-    {
-        private static int InducedGCIterations = 10;
```

Comment:
I'm not sure we have an equivalent test for the moder infrastructure.  We may want to add one, but that's not part of this PR.